manifests/deis-minio-rc.yaml

```diff
@@ -15,11 +15,27 @@ spec:
         app: deis-minio
     spec:
       containers:
-        - imagePullPolicy: Always
-          name: deis-minio
-          image: quay.io/deisci/minio:v2-beta
+        - name: deis-minio
+          image: quay.io/arschles/minio:devel
+          imagePullPolicy: Always
+          env:
+            - name: HEALTH_SERVER_PORT
+              value: "8095"
```

Comment:
Ah, thanks. I want to be explicit here, even if the specified port specified is the default. I'll fix the mismatch though!